apps/i18n/common/en_us.json

```diff
@@ -150,12 +150,14 @@
   "aiTutor": "AI Tutor",
   "aiUserMessagePlaceholder": "Add a chat message...",
   "aiWarningModalHeader": "Remember to chat responsibly!",
-  "aiWarningModalInappropriateFlagged": "Inappropriate messages will be flagged and your teacher will be notified.",
-  "aiWarningModalMessagesAreRecorded": "All messages you send are recorded.",
+  "aiWarningModalInappropriateFlagged": "Inappropriate messages will be flagged by our content moderation policy.",
+  "aiWarningModalMessagesAreRecorded": "All messages you send are recorded and visible to your teacher.",
   "aiWarningModalPersonalNotSubmitted": "In order to protect your privacy, anything considered personal information (name, phone number, address, etc.) should not be submitted.",
   "aiWarningModalOk": "Start chatting",
+  "aiWarningUnsaved": "All unsaved changes will be lost.",
   "aiChatBotIconAlt": "An icon depicting a robot",
-  "aiChatInappropriateUserMessage": "This message has been flagged as inappropriate.",
+  "aiChatInappropriateUserMessage": "This message has been flagged by our content moderation policy. Please try a different message.",
+  "aiChatInappropriateModelMessage": "The Chatbot response has been flagged by our content moderation policy. Please try a different message",
```

Comment:
@samantha-code just confirming you wanted the 'C' in 'The Chatbot response' to be uppercase.

Reply:
Just confirmed with @samantha-code via Slack that she didn't intend to capitalize.